rollup.config.js

```diff
@@ -0,0 +1,43 @@
+import babel from 'rollup-plugin-babel';
+import commonjs from 'rollup-plugin-commonjs';
+import resolve from 'rollup-plugin-node-resolve';
+import { terser } from "rollup-plugin-terser";
+
+export default [
```

Comment:
this rollup config produces two artifacts:

1. `dist/testTrack.js` which is an ES6 module version of the library, it treats jquery (and crypto which is provided by the browser) as external, so as not to bloat the size. The other dependencies, like uuid and md5 are bundled cuz they are specific to this library and pretty small.

2. `dist/testTrack.bundle.js` which is equivalent to the current artifact. its a UMD style JS file, that is minified and uglified. it also does not include jquery.